app/controllers/course/assessment/assessments_controller.rb

```diff
@@ -82,7 +82,9 @@ def load_category
                   current_course.assessment_categories.new.tap do |c|
                     c.assign_attributes(title: t('course.assessment.assessments.index.header'),
                                         weight: 0)
-                    c.tabs.build(title: t('course.assessment.assessments.index.header'), weight: 0)
+                    c.tabs.build(title: t('course.assessment.assessments.index.header'),
+                                 weight: 0,
+                                 category: c)
```

Comment:
I was thinking an `after_create` in `Course`, i.e. making them part of the initialisation of a course, since a course always needs at least one of each for assessments.

I guess we should also prevent the last tab and category from being deleted?